persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/PersistentEntityRegistry.scala

```diff
@@ -38,6 +38,16 @@ trait PersistentEntityRegistry {
    * all persistent events of all `Order` entities.
    */
   def eventStream[Event <: AggregateEvent[Event]](
```

Comment:
This requires careful documentation for the semantics of the offset compared to the `Long` version. Inclusive vs exclusive. See scaladoc here https://github.com/akka/akka-persistence-cassandra/blob/master/src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala#L232 and https://github.com/akka/akka-persistence-cassandra/blob/master/src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala#L276